static/openapi/prerendering-spec.json

```diff
@@ -2059,999 +1891,1491 @@
+                    "status": "OK",
+                    "message": "Request processed",
+                    "data": { "p90": 7355, "p95": 8007, "p99": 10517 }
+                  }
                 }
               }
             }
           },
           "400": {
-            "description": "Bad request",
+            "description": "Bad request.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Bad request",
+                  "description": "Bad request.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Header x-photoniq-customerid is required"
+                  }
                 }
               }
             }
           },
           "500": {
-            "description": "Internal server error",
+            "description": "Internal server error.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Internal server error",
+                  "description": "Internal server error.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Internal server error"
+                  }
                 }
               }
             }
           }
         }
       }
     },
-    "/api/prerender/v1/origins/{origin}": {
+    "/api/prerender/v1/origins/{origin}/prefetch/configs": {
       "get": {
-        "summary": "Get origin",
-        "tags": ["Origins"],
-        "description": "Returns an origin.",
+        "summary": "Get prefetch configuration",
+        "tags": ["Prefetch"],
+        "description": "Returns the prefetch configuration associated with the origin.",
```

Comment:
Do people know what a "prefetch configuration" is?

Reply:
That's something we will discuss in depth once this PR is merged and I write docs. For now, we have this [Prefetching](https://www.macrometa.com/docs/photoniq/prerendering/how-prerender-works#prefetching) section in the documentation.